module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <array>
+#include <vector>
+
+constexpr size_t width = 32;
+constexpr size_t height = 32;
+constexpr uint8_t ASCIIspace = ' ';
+
```

Comment:
Have you considered aliases for longer types like std::array<std::array<uint8_t, height>, width>?